libs/dh/admin/feature-user-management/src/lib/overview/dh-users-overview-table.component.html

```diff
@@ -20,7 +20,7 @@
     [columns]="columns"
     (rowClick)="onRowClick($event)"
     [activeRow]="activeRow"
-    sortBy="email"
+    [sortBy]="UserOverviewSortProperty.Email"
```

Comment:
issue: noticed that this doesn't work because of capitalization. The `columns` object has an `email` property (starts with small `e`) whereas `UserOverviewSortProperty.Email` value starts with capital `E`.